src/muya/lib/contentState/clickCtrl.js

```diff
@@ -78,6 +78,7 @@ const clickCtrl = ContentState => {
     // format-click
     const node = selection.getSelectionStart()
     const inlineNode = node ? node.closest('.ag-inline-rule') : null
+    const parentNode = inlineNode ? inlineNode.parentNode : null
```

Comment:
only get `parentNode` maybe not enough to find the `A` tag?

Reply:
Yes, let me see if there is any other way.